### PR TITLE
feat: add check-strongswan-connection.rb plugin

### PR DESCRIPTION
**Pull request by nishiki**
## Pull Request Checklist

Hi,

I created a new plugin to check the status for one ipsec connection.
I use this plugin on my production environment, and i wanted to shared it with you

Regards,

#### General

- [X] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [X] RuboCop passes

- [X] Existing tests pass

#### New Plugins

- [ ] Tests

- [X] Add the plugin to the README

- [X] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by majormoses** [comment]
Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- add check-strongswan-connection.rb plugin
```
Comment (majormoses) [inline CHANGELOG.md]: a bit better description would be ideal something like:
```
### Added
check-strongswan-connection.rb: checks the health of a strongwan connection (@nishiki)
```

**Review by majormoses — CHANGES_REQUESTED** [review]
Overall I think this looks great can you enhance the changelog entry and provide a [testing artifact](https://github.com/sensu-plugins/community/blob/master/PULL_REQUEST_PROCESS.md#7-testing-artifacts) which can be a simple showing the script input and output.

**Comment by nishiki** [comment]
Hello,

Thanks for your review.

Below the output of my manual tests
```
# /opt/sensu/embedded/bin/ruby check-strongswan-connection.rb --connection dc-fr-pa2
CheckStrongswanConnection OK: the connection dc-fr-pa2 is up
# /opt/sensu/embedded/bin/ruby check-strongswan-connection.rb --connection dc-fr-th3
CheckStrongswanConnection CRITICAL: the connection dc-fr-th3 is down
# /opt/sensu/embedded/bin/ruby check-strongswan-connection.rb --connection  dc-fr-pa666
CheckStrongswanConnection WARNING: the connection dc-fr-pa666 doesn't exist
```

**Review by majormoses — APPROVED** [review]

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-strongswan/versions/1.1.0

**Comment by nishiki** [comment]
Great, thanks!

**Pull request merged**